### PR TITLE
Add explanation of `--ctor` JSON string

### PR DESCRIPTION
**Pull request by gqqnbig**
#### Type of change

- Documentation update

#### Description

Neither the documentation nor the source code of peer mention the exact format of `--ctor` JSON string. Beginners are likely to frown at that.

By looking at the source and asking questions online, I think I figured out the format of  `--ctor` JSON string. 
In this PR, I add the format explanation on the docs of the peer command.

I am also a newbie. Please check if the explanation is correct and revise.

### CODE REVIEW

File: docs/source/commands/peerchaincode.md [hunk 1]
```diff
@@ -28,6 +28,18 @@ different chaincode operations that are relevant to a peer. For example, use the
...
+that has either key 'Args' or 'Function' and 'Args'. These keys are 
+case-insensitive.
+
+If the JSON string only has the Args key, the key value is an array, where the
+first array element is the target function to call, and the subsequent elements
+are arguments of the function. If the JSON string has both 'Function' and
+'Args', the value of Function is the target function to call, and the value of
+Args is an array of arguments of the function. For instance,
+`{"Args":["GetAllAssets"]}` is equvalent to
+`{"function":"GetAllAssets", "Args":[]}`.
```
Comment (denyeart) [inline docs/source/commands/peerchaincode.md]:
```suggestion
`{"Function":"GetAllAssets", "Args":[]}`.
```

**Review by denyeart — COMMENTED** [review]
Newbie improvements are always welcome since they often find things the experienced people overlook, like this one!

It looks correct to me... maybe @mbwhite can look as well.

**Comment by denyeart** [comment]
Actually, the build failed because this doc is auto-generated from other files and the generated command help. Specifically, the text being edited here needs to be updated in docs/wrappers/peer_chaincode_preamble.md, and then peerchaincode.md will get auto-generated from that in the build.

@gqqnbig Could you give that a try?

**Comment by gqqnbig** [comment]
@denyeart I think I fixed it. Can you check?

File: docs/source/commands/peerchaincode.md [hunk 2]
```diff
@@ -28,6 +28,18 @@ different chaincode operations that are relevant to a peer. For example, use the
...
+Some subcommands take flag `--ctor`, of which the value must be a JSON string 
+that has either key 'Args' or 'Function' and 'Args'. These keys are 
+case-insensitive.
+
+If the JSON string only has the Args key, the key value is an array, where the
+first array element is the target function to call, and the subsequent elements
+are arguments of the function. If the JSON string has both 'Function' and
+'Args', the value of Function is the target function to call, and the value of
+Args is an array of arguments of the function. For instance,
+`{"Args":["GetAllAssets"]}` is equvalent to
```
Comment (denyeart) [inline docs/source/commands/peerchaincode.md]:
```suggestion
`{"Args":["GetAllAssets"]}` is equivalent to
```

File: docs/wrappers/peer_chaincode_preamble.md [hunk 3]
```diff
@@ -23,6 +23,18 @@ different chaincode operations that are relevant to a peer. For example, use the
...
+Some subcommands take flag `--ctor`, of which the value must be a JSON string 
+that has either key 'Args' or 'Function' and 'Args'. These keys are 
+case-insensitive.
+
+If the JSON string only has the Args key, the key value is an array, where the
+first array element is the target function to call, and the subsequent elements
+are arguments of the function. If the JSON string has both 'Function' and
+'Args', the value of Function is the target function to call, and the value of
+Args is an array of arguments of the function. For instance,
+`{"Args":["GetAllAssets"]}` is equvalent to
```
Comment (denyeart) [inline docs/wrappers/peer_chaincode_preamble.md]:
```suggestion
`{"Args":["GetAllAssets"]}` is equivalent to
```

**Review by denyeart — COMMENTED** [review]
Looks good now, but I found one more typo.

**Review by denyeart — APPROVED** [review]
Thank you!

**Comment by nikhil550** [comment]
@mergifyio backport release-2.3

**Comment by nikhil550** [comment]
@mergifyio backport release-2.2

**Pull request merged**